Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.yml

```diff
@@ -79,7 +79,7 @@ configuration:
 - display: Indicator Fetch Limit
   name: limit
   type: 0
-  additionalinfo: Provide value to limit the indicator to be fetched per iteration
+  additionalinfo: Provide value to limit the indicator to be fetched per iteration (Limit higher than 30 is not recommended)
```

Comment:
make sure to add this in the readme as well.

Reply:
Done